dist/css/style.css

```diff
@@ -157,12 +182,43 @@ input::placeholder {
   padding-left: 8px;
 }
 
+/*********  RESULTS MODAL  *********/
+#results-modal h3 {
+  text-align: center;
+}
+
+#difference-comment {
+  text-align: center;
+  color: #a0a0a0;
+}
+
+/* .ReactModal__Content .ReactModal__Content--after-open {
```

Comment:
Do we need these comments later? If not, could you remove them.

Reply:
Keep for now.  Not sure if I need it yet.